### PR TITLE
Adding kotlin bazel tests

### PR DESCRIPTION
**Pull request by deannagarcia**
Adding bazel run tests for both kotlin and kotlin-lite subpackages and adding those tests into the full java test suite.

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -59,3 +59,18 @@ bazel_skylib_workspace()
 
 load("@rules_pkg//:deps.bzl", "rules_pkg_dependencies")
 rules_pkg_dependencies()
+
+# For `kt_jvm_library`
+rules_kotlin_version = "v1.5.0-beta-4"
```
Comment (perezd) [inline WORKSPACE:64]: This belongs in here:
https://github.com/protocolbuffers/protobuf/blob/master/protobuf_deps.bzl

**Review by perezd — CHANGES_REQUESTED** [review]
Overall looks good, just audit your use of public and ensure its more tightly locked down.

File: BUILD [hunk 2]
```diff
@@ -1320,6 +1322,19 @@ cc_binary(
 #     ],
 # )
 
+
+java_proto_library(
+    name = "java_test_protos",
+    deps = [":generic_test_protos"],
+    visibility = ["//visibility:public"],
```
[the remark this answers is not in the document]
Comment (deannagarcia) [reply BUILD]: Changed them all to be restricted to the java subpackage.

File: WORKSPACE (hunk 1 above)
Comment (deannagarcia) [reply WORKSPACE]: Moved the http_archive to protobuf_deps.bzl but I think I have to keep the registering toolchain calls in WORKSPACE, so I left those there.

**Review by perezd — APPROVED** [review]
LGTM!

**Comment by meteorcloudy** [comment]
FYI, looks like those kotlin tests don't work on Windows:
https://buildkite.com/bazel/protobuf/builds/4038#a77575c0-a256-4ddd-9e5b-7410337bc054

**Comment by deannagarcia** [comment]
> FYI, looks like those kotlin tests don't work on Windows: https://buildkite.com/bazel/protobuf/builds/4038#a77575c0-a256-4ddd-9e5b-7410337bc054

Thanks for this report! I'm looking into this now.

**Comment by deannagarcia** [comment]
> > FYI, looks like those kotlin tests don't work on Windows: https://buildkite.com/bazel/protobuf/builds/4038#a77575c0-a256-4ddd-9e5b-7410337bc054
> 
> Thanks for this report! I'm looking into this now.

This should be fixed with: https://github.com/protocolbuffers/protobuf/pull/9428

**Pull request merged**